src/test/functional/functional-tests.js

```diff
@@ -200,32 +224,30 @@ describe('functional tests', function() {
   })
 
   describe('makeBucket with region', () => {
-    step(`makeBucket(bucketName, region, cb)_bucketName:${bucketName}-region, region:us-east-2_`, done => {
-      try {
-        clientUsEastRegion.makeBucket(`${bucketName}-region`, 'us-east-2', assert.fail)
-      } catch (e) {
-        done()
-      }
+    // When custom region is set, these tests are expected to fail.
```

Comment:
I hope I understand this comment correctly, if not apologies for my misunderstanding.
So, if a custom region is set on the server side, these tests will fail. Is that right?
If so, how are these failures going to be reported to the end-user?
If we are going to report these tests as "FAILED" and expect the end-user to interpret the failures as "PASS", that is not good. We should make sure we return "PASS" in all scenarios.